app/main.py

```diff
@@ -1,3 +1,61 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: "int, float") -> "Distance":
```

Comment:
The type annotation for the `other` parameter should be `Union[Distance, int, float]` instead of the string 'int, float'. This will properly indicate that `other` can be a `Distance` instance or a numeric type.